napari/tests/test_viewer.py

```diff
@@ -41,7 +46,16 @@ def test_viewer(qtbot):
         if func.__name__ == 'play':
             func(viewer)
 
-    # Close the viewer
+    if sys.platform == 'darwin':
+        # on some versions of Darwin, exiting while fullscreen seems to tickle some bug deep in NSWindow.  This forces
```

Comment:
```
        # on some versions of Darwin, exiting while fullscreen seems to tickle some bug
        # deep in NSWindow.  This forces the fullscreen keybinding test to complete its
        # draw cycle, then pop back out of fullscreen.
```

Reply:
> can you decrease the line length here?

Done.  Neither black nor the CI caught this FYI.